cosmos/providers/dbt/core/operators.py

```diff
@@ -227,7 +228,9 @@ def run_command(self, cmd, env):
         return result
 
     def build_and_run_cmd(self, env: dict, cmd_flags: list = None):
-        create_default_profiles()
+        create_default_profiles(
+            Path(os.path.expanduser("~")).joinpath(".dbt/profiles.yml")
+        )
```

Comment:
I had a good read of the [docs](https://docs.getdbt.com/docs/get-started/connection-profiles#connecting-to-your-warehouse-using-the-command-line)
If we were to create a constant for where we loaded the file and also set DBT_PROFILES_DIR (or add it as a global flag) then it could get around any complications where users have loaded a profiles.yml in to the root of their dbt project, thereby superseding the Cosmos setup.

Reply:
@MikeWallis42 would we just set that `DBT_PROFILES_DIR` to "~/.dbt/profiles.yml" so that users couldn't override the profiles.yml on their own?

Reply:
@pohek321 seems like a good idea to me.
If we have it as a constant then and add it to `DbtBaseOperator.get_env` as well as make use of it in `create_default_profiles` that should close the gap.
I'll add a commit with the changes for review.